Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jinaai/jina:2.0.0rc9
+FROM jinaai/jina:2.0
```

Comment:
update jina version

Reply:
Updated in https://github.com/jina-ai/executor-sentence-transformer/pull/3/commits/6f331d2e31b15f73dc955cacdf7161fe8c6d6f5d